index.js

```diff
@@ -181,7 +181,8 @@
   //. The Maybe type represents optional values: a value of type `Maybe a` is
   //. either a Just whose value is of type `a` or a Nothing (with no value).
   //.
-  //. The Maybe type satisfies the [Monoid][] and [Monad][] specifications.
+  //. The Maybe type satisfies the [Extend][], [Monoid][], and [Monad][]
+  //. specifications.
```

Comment:
I'd like to list Extend last. If one day we upgrade from Extend to Comonad (which may or may not be possible) it would be strange to list Comonad before Monad. As a bonus, Monoid, Monad, Extend matches the order the type classes appear in the spec.